.helix/languages.toml

```diff
@@ -0,0 +1,2 @@
+[language-server.rust-analyzer.environment]
+DATABASE_URL = "postgresql://postgres:postgres@127.0.0.1:5432"
```

Comment:
Could you add .helix to .gitignore?

Reply:
Is there a reason we can't have this just work out of the box for helix users?

Reply:
There are multiple editors and multiple people using it. I personally use Neovim and my colleges use VSCode and Emacs. If we added config files for all editors, we'd have tons of folders and files that are not related to the code.

We usually use `.env` to set DATABASE_URL for our development, which is ignored, since there's no standard way to run the test database.  I personally use port 5433, so this would be an issue for me even if I used helix.